fastlane/spec/fixtures/device_grid_results.html

```diff
@@ -20,7 +20,7 @@
 <td>
 <a href='https://appetize.io/embed/1461233806?autoplay=true&orientation=portrait&device=iphone4s&deviceColor=black&scale=75&language=de'>
 <p align='center'>
-<img height='130' src='https://raw.githubusercontent.com/fastlane/fastlane/master/fastlane/lib/fastlane/actions/device_grid/assets/iphone4s.png' />
+<img height='130' src='https://raw.githubusercontent.com/fastlane/fastlane/1.81.0/fastlane/lib/fastlane/actions/device_grid/assets/iphone4s.png' />
```

Comment:
This test will break the next time we bump fastlane versions.  I'd mock the call to to Fastlane::VERSION in the spec.